@NixOS/nixpkgs-vet nixpkgs-vet seems to be failing to build itself:

```
building '/nix/store/1sm6871f3k4j7malkz491bhg0q79162y-nixpkgs-vet.drv'...
error: creating symlink '/build/.local/share/nix/root/nix/var/nix/gcroots/profiles' -> '/build/.local/share/nix/root/nix/var/nix/profiles': File exists
- Nix evaluation failed for some package in `pkgs/by-name`, see error above
This PR introduces the problems listed above. Please fix them before merging, otherwise the base branch would break.
error: builder for '/nix/store/1sm6871f3k4j7malkz491bhg0q79162y-nixpkgs-vet.drv' failed with exit code 1;
       last 3 log lines:
       > error: creating symlink '/build/.local/share/nix/root/nix/var/nix/gcroots/profiles' -> '/build/.local/share/nix/root/nix/var/nix/profiles': File exists
       > - Nix evaluation failed for some package in `pkgs/by-name`, see error above
       > This PR introduces the problems listed above. Please fix them before merging, otherwise the base branch would break.
       For full logs, run:
         nix log /nix/store/1sm6871f3k4j7malkz491bhg0q79162y-nixpkgs-vet.drv
To run locally: ./ci/nixpkgs-vet.sh master https://github.com/NixOS/nixpkgs.git
If you're having trouble, ping @NixOS/nixpkgs-vet
Error: Process completed with exit code 1.
```

--- a/pkgs/by-name/gp/gpredict-unstable/package.nix
+++ b/pkgs/by-name/gp/gpredict-unstable/package.nix
@@ -11,7 +11,7 @@
 }).overrideAttrs
   (finalAttrs: {
     # Next version is 2.4.0
-    version = "2.4.0-unstable-2024-09-17";
+    version = "2.2.1-unstable-2024-09-17";
 
     src = fetchFromGitHub {
       owner = "csete";

--- a/pkgs/by-name/gp/gpredict-unstable/package.nix
+++ b/pkgs/by-name/gp/gpredict-unstable/package.nix
@@ -10,7 +10,6 @@
   goocanvas2 = goocanvas3;
 }).overrideAttrs
   (finalAttrs: {
-    # Next version is 2.4.0
     version = "2.2.1-unstable-2024-09-17";
 
     src = fetchFromGitHub {

